client/my-sites/domains/domain-management/settings/cards/domain-redirect-card.tsx

```diff
@@ -221,46 +243,16 @@ const DomainRedirectCard = ( props: DomainRedirectCardProps & PropsFromRedux ) =
 				<FormButton
 					disabled={
 						! isValidUrl ||
-						isFetching ||
-						isUpdating ||
-						( target === targetUrl && ( redirect?.isSecure ? 'https' : 'http' ) === protocol )
+						isLoading ||
+						( redirect &&
+							redirect.targetHost + redirect.targetPath === targetUrl &&
+							( redirect.isSecure ? 'https' : 'http' ) === protocol ) ||
+						( ! redirect && targetUrl === '' )
```

Comment:
When we add more to this form we should replace this with some kind of hasChanges state check. For now it works well enough :grimacing: